File: Proxy/Java/Cuenta.java
public class Cuenta {

    private int idCuenta;
    private String usuario;
    private double saldoInicial;

    public Cuenta(int idCuenta, String usuario, double saldoInicial){
        this.idCuenta = idCuenta;
        this.usuario = usuario;
        this.saldoInicial = saldoInicial;
    }
    
    public int getIdCuenta(){ return this.idCuenta; }
    public String getUsuario(){ return this.usuario; } 
    public double getSaldoInicial(){ return this.saldoInicial; }

    public void setIdCuenta(int idCuenta){this.idCuenta = idCuenta;}
    public void setUsuario(String usuario){this.usuario = usuario;}
    public void setSaldoInicial(double saldoInicial){this.saldoInicial = saldoInicial;}
}
 

File: Proxy/Python/Cuenta.py
class Cuenta:

    def __init__(self, idcuenta, usuario, saldoInicial):
        self._idCuenta = idcuenta
        self._usuario = usuario
        self._saldoInicial = saldoInicial
    
    # Getters
    def getCuenta(self):
        return self._idCuenta
    
    def getUsuario(self):
        return self._usuario
        
    def getSaldoInicial(self):
        return self._saldoInicial
    
    # Setters
    def setIdCuenta(self, idCuenta):
        self._idCuenta = idCuenta
    def setUsuario(self, usuario):
        self._usuario = usuario
    def setSaldoInicial(self, saldoInicial):
        self._saldoInicial = saldoInicial